### PR TITLE
Add main controller tests for a few extensions

### PR DESCRIPTION
**Pull request by kisantia**
A little cleanup and adding a few small tests so that the mainController and extension files get some coverage for dacpac, schema compare, and sql database projects

### CODE REVIEW

File: extensions/sql-database-projects/src/test/mainController.test.ts [hunk 1]
```diff
@@ -18,6 +18,11 @@ import { shouldThrowSpecificError } from './testUtils';
 
 let testContext: TestContext;
 
+function createController(): MainController {
```
Comment (Charles-Gagnon) [inline extensions/sql-database-projects/src/test/mainController.test.ts:21]: Why did you create a separate function for this? It's just a single line as it is so doesn't really seem valuable

Comment (kisantia) [reply extensions/sql-database-projects/src/test/mainController.test.ts:21]: I was just copying what the ML extension did, but I'll get rid of it

**Comment by kisantia** [comment]
@Charles-Gagnon is it better to register commands in extension.ts? Both schema compare and dacpac extensions don't do anything besides register their command in MainController, so those can go away if registering commands moves to extension.ts.

**Comment by Charles-Gagnon** [comment]
I don't really have a huge preference - but I don't see any reason why it shouldn't be. General extension setup stuff I think is reasonable to all keep in extension.ts unless it gets really complex

**Pull request merged**